internal/bminventory/inventory.go

```diff
@@ -4688,7 +4671,7 @@ func (b *bareMetalInventory) UpdateHostInstallProgress(ctx context.Context, para
 
 		log.Info(fmt.Sprintf("Host %s in cluster %s: %s", host.ID, host.ClusterID, event))
 		msg := fmt.Sprintf("Host %s: %s", hostutil.GetHostnameForMsg(&host.Host), event)
-		b.eventsHandler.AddEvent(ctx, *host.ClusterID, host.ID, models.EventSeverityInfo, msg, time.Now())
+		eventgen.SendGenericHostEvent(ctx, b.eventsHandler, *host.ID, *host.ClusterID, msg, models.EventSeverityInfo)
```

Comment:
the fourth argument should be infra_env_id

Reply:
As mentioned here https://github.com/openshift/assisted-service/pull/2717#discussion_r727761899, since it's a V1 function, we assume here that `infra-env-id = cluster-id`